lib/index.js

```diff
@@ -56,7 +58,31 @@ Vero.prototype.group = request('/users/edit', 'put');
  * @api public
  */
 
-Vero.prototype.track = request('/events/track');
+Vero.prototype.track = function(payload, fn) {
+  if (payload.event().match(/[Uu]nsubscribe/)) {
```

Comment:
If you don't need the matches, `RegExp.prototype.test` is much more efficient than `RegExp.prototype.match`